tools/dify_hugegraph/test_hugegraph.py

```
@@ -0,0 +1,75 @@
+        # 'username': 'admin',
+        # 'password': 'admin'
+    }
+
+    print("=== Testing HugeGraph Dify Tool Plugin ===\n")
+
+    # Test 1: Get schema (should work even if HugeGraph is not running)
+    print("1. Testing get_schema...")
+    try:
+        result = tool.get_schema(test_params)
+        print(f"Result: {result[:200]}..." if len(result) > 200 else f"Result: {result}")
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)